src/userCLI/build.gradle

```diff
@@ -9,20 +9,8 @@ buildscript {
     }
 }
 
-plugins {
```

Comment:
Again all the deleted code here has just moved to the root project "subprojects" block